Common/src/CConfig.cpp

```diff
@@ -629,6 +630,45 @@ unsigned short CConfig::GetnZone(const string& val_mesh_filename, unsigned short
 
     }
 
+    case SU2_BIN: {
+
+      /*--- Check if the mesh file can be opened for binary reading. ---*/
+      FILE *mesh_file = fopen(val_mesh_filename.c_str(), "rb");
+      if ( !mesh_file )
+        SU2_MPI::Error(string("There is no geometry file called ") + val_mesh_filename,
```

Comment:
I don't think you need to cast to string explicitly in these places
```suggestion
        SU2_MPI::Error("There is no geometry file called " + val_mesh_filename,
```